### PR TITLE
Added morgan for request logging and winston for internal logging.

### PR DESCRIPTION
**Pull request by thaigillespie**

### CODE REVIEW

File: src/logging.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import winston from 'winston';
+import dotenv from 'dotenv';
+
+dotenv.config();
```
Comment (godwinpang) [inline src/logging.ts:4]: dotenv.config should be called only once - the idea is that we have a config in config.ts and call it there. 
from there on out, nothing else will directly access process env. they will instead access the config

Comment (thaigillespie) [reply src/logging.ts:4]: Shoot I used dotenv.config() outside of config.js in a couple more places in the codebase. I'll remove them in this PR then?

Comment (godwinpang) [reply src/logging.ts:4]: it's ok - do it after postgres merges in

**Comment by godwinpang** [comment]
future todo is to reference: https://github.com/UCSDTESC/Check-in/blob/master/src/server/config/Logging.ts

**Pull request merged**